sys/cddl/dev/dtrace/dtrace_ioctl_compat.c

```diff
@@ -0,0 +1,260 @@
+/*
+ * CDDL HEADER START
```

Comment:
Is any of this code copied from CDDL sources or is it all your code?  If it's all yours, it should likely be BSD licensed like much of the other code in this directory (although that's your choice).

Reply:
Fixed